website/docs/diff.md

```diff
@@ -15,6 +15,8 @@ sidebar_position: 6
    * database name cannot contain capital letters.
 5. For Tigris, FerretDB requires Tigris schema validation for `msg_create`: validator must be set as `$tigrisSchemaString`.
    The value must be a JSON string representing JSON schema in [Tigris format](https://docs.tigrisdata.com/overview/schema).
+6. For Tigris, if a field is represented as a number, any attempt to assign a string containing a number will pass as a number
```

Comment:
⚠️

The idea of that list of user-visible differences was to have a corresponding dance test for each item. Are we going to add that?

I would say we should not bother with it until we add MongoDB-style schema validation, so let's remove that item for now.

Reply:
> The idea of that list of user-visible differences was to have a corresponding dance test for each item

Actually, please add a markdown comment above that list mentioning that

Reply:
@AlekSi sorry, I don't understand what should be mentioned where. Could you suggest a change, please?
If `dance` supports Tigris, let's add point 6, yes.

Reply:
fixed